scripts/calibrateArmDac.sh

```diff
@@ -1,26 +1,51 @@
-#!/bin/zsh
+#!/bin/bash
 # Usage:
-#   executeScurveProgram.sh <DetName> <cardName> <link> <vfatmask> <Comma Separated List of CFG_THR_ARM_DAC Values>
+#   calibrateArmDac.sh <DetName> <cardName> <link> <vfatmask> <Comma Separated List of CFG_THR_ARM_DAC Values>
 
 helpstring="Usage:
-    ./executeScurveProgram.sh [DetName] [cardName] [link] [vfatmask] [comma separated list of CFG_THR_ARM_DAC Values]
+    ./calibrateArmDac.sh [DetName] [cardName] [shelf]  [slot] [link] [vfatmask] [comma separated list of CFG_THR_ARM_DAC Values]
+    ./calibrateArmDac.sh -r [PATH] [DetName] [cardName] [shelf] [slot] [link] [vfatmask] [comma separated list of CFG_THR_ARM_DAC Values]
```

Comment:
I think `cardName` should be dropped from both the usage example and also the `helpstring`.

Reply:
done